<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublicDatasetProject</class>
 <widget class="QWidget" name="PublicDatasetProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>203</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Public Dataset Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>media/logo.png</normaloff>media/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="PublicDatasetProjectLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="PublicDatasetProjectGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <layout class="QHBoxLayout" name="PublicDatasetProjectGroupHLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="PublicDatasetProjectGroupLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="ProjectLayout">
          <item>
           <widget class="QLabel" name="ProjectLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Project</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ProjectConfig">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="WorkerLayout">
          <item>
           <widget class="QLabel" name="WorkerLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Worker</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="WorkerConfig">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="LocationLayout">
          <item>
           <widget class="QLabel" name="LocationLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="LocationPathLayout" stretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="LocationPathDisplay">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>575</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="LocationPathSelection">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="folder-open"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ModelLayout">
          <item>
           <widget class="QLabel" name="ModelLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Model</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ModelSelectionLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="ModelTypeSelection">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>AnimalRTPose</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>AnimalViTPose</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SPIPose</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YOLO12-Pose</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YOLO11-Pose</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YOLOv8-Pose</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YOLOv8-Pose-P6</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ModelScaleSelection">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>N</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>S</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>M</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>L</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>H</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DatasetLayout">
          <item>
           <widget class="QLabel" name="OthersLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Dataset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="DatasetSelection">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>AcinoSet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AnimalKingdom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AnimalPose</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AniposeFly</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AnimalMouse</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AP10K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>APT36K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>APTv2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ATRW</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AwA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>COCO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fish</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fly</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Horse10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Locust</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LoTE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LSP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Macaque</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Marmoset</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OpenMonkeyChallenge</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pups</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Quadruped-80K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>StanfordExtra</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TopViewMouse</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tri-Mouse</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Zebra</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CreateProjectBase">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>345</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Create Public Dataset Project</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
